src/test/run-make/yk-rustc-hwtracer/Makefile

```diff
@@ -0,0 +1,8 @@
+-include ../../run-make-fulldeps/tools.mk
+
+all:
+	$(RUSTC) -C tracer=hw block.rs 
+	[ "`dwarfdump ${TMPDIR}/block | grep __YK_BLK | wc -l`" != 0 ]
```

Comment:
Why do we have one `!=` and one `-eq`. I suppose you want to consistently use the numeric variant?

Reply:
It seems that if I change this the tests fail (see commits).